pandas/core/arrays/datetimes.py

```diff
@@ -253,28 +257,22 @@ def _generate_range(cls, start, end, periods, freq, tz=None,
 
         if tz is not None:
             # Localize the start and end arguments
-            start = _maybe_localize_point(
-                start, getattr(start, 'tz', None), start, freq, tz
-            )
-            end = _maybe_localize_point(
-                end, getattr(end, 'tz', None), end, freq, tz
-            )
+            start = _maybe_localize_point(start, getattr(start, 'tz', None),
+                                          start, freq, tz)
```

Comment:
In general, can you leave such style changes only to lines you actually change anyway?

Reply:
yeah and the former is actually more idiomatic, really prefer not to do partial line wrapping

Reply:
sure